sapl/sessao/forms.py

```diff
@@ -349,14 +349,6 @@ def clean(self):
         else:
             cleaned_data['materia'] = materia
 
-        exists = self._model.objects.filter(
```

Comment:
O quão complexo seria emitir uma modal falando que a matéria já existe e perguntando se deseja continuar? 

A gente já tem modais assim quando o ano da norma é diferente da data e agora acabei de fazer o merge de um PR do Cantuária que pergunta quando se muda na norma a matéria referenciada. Mas como essa tela de sessão é meio enrolada se for muito dificil podemos simplesmente deixar assim. Teria como ver essa possibilidade?

Reply:
Alterado. PR #3224.